### DISCUSSION TITLE
Highlight relationship when related column is selected

### DISCUSSION THREAD

**Discussion by MarioMue13** [TabularEditor/TabularEditor3#1130] [Ideas]
In Big models it makes it easier to check if a relationship already exists

**Comment by otykier** [comment]
Hi @MarioMue13 
Are you referring to highlighting in the TOM Explorer?
Because we already do highlight relationships in the diagram view, when selecting a column (and vice versa)

![diagram-highlight](https://user-images.githubusercontent.com/8976200/228842978-9e3f6af8-00af-4eef-b32c-2cde64f1131f.gif)

**Comment by MarioMue13** [comment]
yes Tom Explorer. I usuallay work without the diagram view

![image](https://user-images.githubusercontent.com/74975641/228907497-3574b4bf-8d00-441b-a76a-a6c1fdfa986e.png)

**Comment by edhans** [comment]
If I could tag along on this, it would be nice if when you right-clicked on a field that was used in a relationship and viewed Dependencies, the relationship itself would show up in the dependency view.

I know there is an option in that view to "Show relationships starting from this table" but you have to select the table to see that, not the key itself.

**Comment by DBojsen** [comment]
Hi @MarioMue13 & @edhans 

Thank you for the suggestion and explanation.
We'll take this into concideration - I cannot promise anything right now.

BR David - TE3 Support

**Comment by mlonsk** [comment]
The highlight enhancement as well as the view dependency on columns change is available in Tabular Editor 3.13.0.
Closing the feature request 😁🔏
